pkg/apis/core/types.go

```diff
@@ -17,6 +17,7 @@ limitations under the License.
 package core
 
 import (
+	apiv1 "k8s.io/api/core/v1"
```

Comment:
my apologies... I didn't notice this before. the internal API should not have a package dependency on a particular external API. Use the `PreemptionPolicy` type defined in this file rather than `apiv1.PreemptionPolicy`